po4a-create-all-targets.sh

```diff
@@ -48,14 +48,10 @@ fi
 # REMOVE .md FILE FOLDERS BEFORE REGENERATING THEM
 ##################################################
 
-while IFS= read -r -d '' dir
-do
-	lang=$(basename -s .md "$dir")
-	echo "delete $lang folder"
-	cd "$PUB_DIR"
-	rm -rf "$lang" 
-	cd ../  
-done <   <(find "$PO_DIR" -mindepth 1 -maxdepth 1 -type d -print0)
+for lang in $(ls "$PO_DIR" ) ; do
```

Comment:
Wouldn’t this delete everything (=all file) in $PO_DIR? 

You could then just `rm -rf *` I‘m a bit worried here.

Reply:
No, it takes the directory names in PO_DIR as a reference to delete them in /wiki. That way it only deletes supported language folders (if they have a .po folder they're supported).

Reply:
Yes. You’re running rm -r on $PUB_DIR